templates/deceasedreports/review.html.twig

```diff
@@ -5,6 +5,8 @@
         <h2><i class="fa fa-hourglass-o" aria-hidden="true"></i> Deceased Participant <small><a href="{{ path('participant', {id: participant.participantId} )}}">{{ participant.lastName }}, {{ participant.firstName }}</a></small></h2>
     </div>
 
+    <a href="{{ path('deceased_report_history', { participantId : participant.participantId  }) }}">Return to Deceased Participants</a>
```

Comment:
Is this supposed to go back to the history for a participant? Or the main deceased reports list (`deceased_reports_index`)?

Reply:
Yeah, I think it's`deceased_reports_index `